activitypub.php

```diff
@@ -50,7 +50,7 @@
  * Initialize REST routes.
  */
 function rest_init() {
-	Rest\Users::init();
+	Rest\Actors::init();
```

Comment:
So it did happen after all! :) I personally think this is a really welcome change.

Reply:
this change is dedicated to you! :)